birdhouse/components/stac/config/magpie/docker-compose-extra.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.4"
+services:
+  magpie:
+    volumes:
+    - ./components/stac/config/magpie/config.yml:/opt/local/src/magpie/config/permissions/stac.yml:ro
```

Comment:
Please use the `MAGPIE_PERMISSIONS_CONFIG_PATH` and `MAGPIE_PROVIDERS_CONFIG_PATH` environment variables here instead of hard-coding the paths.